src/grt/src/fastroute/src/FastRoute.cpp

```diff
@@ -860,7 +862,6 @@ NetRouteMap FastRouteCore::run()
     VIA = 0;
     THRESH_M = 0;
     CSTEP1 = 30;
-    slope = BIG_INT;
```

Comment:
What error requires this removal?

Reply:
Unused value: assigned_value: Assigning value 10000000 to slope here, but that stored value is overwritten before it can be used.